scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -21,20 +21,26 @@ This notebook will demonstrate how to:
 ---
 
 In this notebook, we'll perform integration on scRNA-seq datasets from the [Single-cell Pediatric Cancer Atlas (`ScPCA`)](https://scpca.alexslemonade.org/), a database of uniformly-processed pediatric scRNA-seq data built and maintained by the Data Lab.
-This database currently hosts projects from ten different ALSF-funded labs who have generated single-cell pediatric cancer transcriptomic data with the ultimate goal of making this data easily accessible to investigators (like you!).
+This database currently hosts (as of March 2023) eleven projects from eight different ALSF-funded labs who have generated single-cell pediatric cancer transcriptomic data with the ultimate goal of making this data easily accessible to investigators (like you!).
+`ScPCA` raw data is processed with [nextflow pipeline](https://github.com/AlexsLemonade/scpca-nf) that uses [`alevin-fry`](https://doi.org/10.1038/s41592-022-01408-3) for fast and efficient mapping, followed by filtering and normalization with Bioconductor tools. 
+You can read more about how data in the `ScPCA` is processed in [the associated documentation](https://scpca.readthedocs.io/en/latest/).
```

Comment:
I was really looking for both more and less here... I was mostly hoping to just get across that we used the most of same tools that they have seen in the first notebook. I added some of that detail below, but check that it is correct!

I also don't think the stats are needed here, especially if we have to keep updating them, so I went the opposite direction and cut the numbers. I also realized the "ultimate goal" part was ambiguous (who's goal?). So I tried to rephrase a bit.

```suggestion
The `ScPCA` database currently hosts single-cell pediatric cancer transcriptomic data generated by ALSF-funded labs, with the ultimate goal of making this data easily accessible to investigators (like you!).
The expression data in `ScPCA` were mapping and quantified with [`alevin-fry`](https://doi.org/10.1038/s41592-022-01408-3), followed by processing with Bioconductor tools, following the same general procedures that we have covered in this workshop.
The raw counts matrix was filtered with `emptyDropsCellRanger()` and `miQC`, log-normalized with `scuttle`, with dimension reduction by `scater`.
The processed data is stored as `.rds` files containing `SingleCellExperiment` objects.
You can read more about how data in the `ScPCA` is processed in [the associated documentation](https://scpca.readthedocs.io/en/latest/).
```